src/inc/corcompile.h

```diff
@@ -727,6 +727,9 @@ enum CORCOMPILE_FIXUP_BLOB_KIND
     ENCODE_CHECK_FIELD_OFFSET,
 
     ENCODE_DELEGATE_CTOR,
+
+    ENCODE_GENERIC_DICTIONARY_ACCESS,
```

Comment:
Also, there should be a matching constant `READYTORUN_FIXUP_XXX` added to readytorun.h and `static_assert_no_msg` to src\zap\zapreadytorun.cpp that verify that the two are in sync.